packages/fxa-settings/src/pages/Signin/SigninTokenCode/en.ftl

```diff
@@ -15,3 +15,5 @@ signin-token-code-code-expired = Code expired?
 # Link to resend a new code to the user's email.
 signin-token-code-resend-code-link = Email new code.
 signin-token-code-required-error = Confirmation code required
+# Error displayed in a tooltip when the form is submitted without a code
+signin-token-code-required-error = Confirmation code required
```

Comment:
Looks like this is a dupe ID/value of L17 (but with a comment). Probably just delete L17.